tests/testthat/test-create-validation-docs.R

```diff
@@ -205,3 +205,41 @@ test_that("create_validation_docs() can auto-assign test IDs", {
                 label = paste0("IDs in ", .f))
   }
 })
+
+test_that("create_validation_docs() drops orphan test IDs from testing docs", {
+  output_dir <- file.path(tempdir(), "mrgvalidate-create-validation-docs")
```

Comment:
This is mostly unrelated, but we should really write a helper to do these four lines because they're all over the place (and not just in this package either). Feels like it should be called `tempdir()` but I guess that's already taken.

Reply:
the helper can also use `withr::defer()` (similar to [this guy](https://github.com/metrumresearchgroup/bbr/blob/develop/tests/testthat/setup-workflow-ref.R#L236-L266) that Sean wrote for `bbr`) instead of `on.exit()` so that it cleans up when the calling function (or any other specified environment) exits.